### PR TITLE
added documentation, updated spliting functionality, which did not work

### PR DESCRIPTION
**Pull request by mikolayek**

### CODE REVIEW

File: crc-scripts-executor.groovy [hunk 1]
```diff
@@ -3,11 +3,13 @@ import org.apache.commons.lang.StringUtils
 import groovy.json.JsonOutput
 import groovy.json.JsonSlurper
 
+// dependencies are available in SAP Commerce environment
+// please run script in hac groovy console
 
 def baseUrl = 'raw.githubusercontent.com/sap-onestrike/crc-scripts/main/scripts'
 
 def includeScripts = []
-def scriptsList = StringUtils.split(IOUtils.toString(new URL("https://${baseUrl}/list.txt?${System.currentTimeMillis()}"), 'utf-8'), '\n')
```
Comment (sap-onestrike) [inline crc-scripts-executor.groovy:10]: Could you please also remove unused imports?

Comment (mikolayek) [reply crc-scripts-executor.groovy:10]: done

Comment (sap-onestrike) [reply crc-scripts-executor.groovy:10]: It was more about your change - removal of `StringUtils` usage, that one import is no longer needed here.

Comment (mikolayek) [reply crc-scripts-executor.groovy:10]: oh. yes. I did not notice that. done

**Review by sap-onestrike — APPROVED** [review]

**Pull request merged**